files/internals/cports.common

```diff
@@ -89,27 +175,57 @@ PROTO="icmp"
 if [ "$ICMP_LIM" == "" ]; then
 	ICMP_LIM=0
 fi
-if [ "$(echo $ICMP_LIM | tr '/' ' ' | awk '{print$1}')" -gt "0" ]; then
-	ICMP_EARGS="-m limit --limit $ICMP_LIM"
+if [ $EXECUTOR == "nft" ]; then
+   if [ "$(echo $ICMP_LIM | tr '/' ' ' | awk '{print$1}')" -gt "0" ]; then
+	ICMP_EARGS="limit rate $ICMP_LIM burst 5 packets"
```

Comment:
ICMP_LIM need to auto convert, example valid command

```
 nft add rule ip6 filter INPUT meta l4proto icmp ip6 daddr $VNET limit rate 5/second burst 5 packets  counter accept
```

Reply:
already https://github.com/jasonwee/advanced-policy-firewall/blob/39a094a17e20885d484e20fd35b86d8c4537feb3/files/internals/internals.conf#L93-L98